app/views/templates/general_content.html.haml

```diff
@@ -1,8 +1,17 @@
+- if node.short_summary
```

Comment:
You are already checking to see if there is a short summary though aren't you? The conditional above.